### ISSUE TITLE
ClearSky server start

### ISSUE THREAD

**Issue by mikelees2** [deotern/Plugswork#10]
### Issue description

ClearSky ver 1.1
<[Server thread/ERROR]: Could not load plugin 'Plugswork': Incompatible API version

### Steps to reproduce the issue

<server start>
### OS and versions

- Plugswork: 3
- PHP: 5
- OS: Linux

**Comment by NIIP99** [comment]
Can you tell me the API Version of the ClearSky u installed? :)

**Comment by mikelees2** [comment]
I got this from the Jenjins site its the latest. I’m researching switching from IM because they are not working on it any more. 
At this point I don’t think this is going to work for me.  Do you have any recommendations for  PE server software to replace the out going IM software?

ClearSky version 1.1 "Sunrise" (API 1.13.1) Build 467
API 1.13.1

**Comment by remotevase** [comment]
@mikelees2 yes we are. https://github.com/Inactive-to-Reactive/ImagicalMine. That is the new official repo for work

**Comment by mikelees2** [comment]
Thanks
where are the phar files located? ( InagicalMine.phar )

**Comment by remotevase** [comment]
That has not been set up yet. You will be able to see updates in this thread when it is set up
https://forums.imagicalmine.net/threads/no-imagicalmine-is-not-gone.1376/#post-13920

**Comment by Aericio** [comment]
API 1.13.1 = PHP5

**Comment by Aericio** [comment]
ClearSky has a php7 version now

**Comment by mikelees2** [comment]
Is it for 15?

**Comment by Aericio** [comment]
Yes, there is a [0.15.0](https://github.com/ClearSkyTeam/ClearSky/tree/php7-0.15) one, but I think its not really tested... you have to compile it your own right now though.

**Comment by Aericio** [comment]
Theres a 0.15.0 ClearSky now. Its functional and is PHP7. It's in PHP7-0.15 branch.
